Added use volume keys to digitize (#381)
opengisch/QField-docs: documentation/how-to/digitize.en.md

    QField insures that digitized geometries will not have duplicate vertices and respects
    the geometry precision settings from the currently selected layer.

> You have the option to enable or disable the functionality that allows you to use the volume keys for adding or removing vertices while in digitizing mode. You can activate this functionality in Settings in the General tab.

Reviewer: ```suggestion
You can use the volume keys for adding or removing vertices while in digitizing mode. This functionality can be activated in Settings in the General tab.
```

Too much suspense before we get to the point.

!![](../assets/images/activate-digitize-with-volumen-keys.png)

!!! note

>     This feature is available on Android.

Reviewer: ```suggestion
    This feature is available on Android only.
```
